### ISSUE TITLE
About "Validation" errors in yml

### ISSUE THREAD

**Issue by animeojisan** [the-database/traiNNer-redux#19]
I used it on Windows.
It seems like there's an extra space here.

```
#############
# Validation
#############
# Uncomment these for validation.
# val:
#   val_freq: 1000
#   save_img: True

#   metrics:
#     psnr:
#       type: calculate_psnr
#       crop_border: 4
#       test_y_channel: false
#     ssim:
#       type: calculate_ssim
#       crop_border: 4  # Whether to crop border during validation.
#       test_y_channel: false  # Whether to convert to Y(CbCr) for validation.
#     dists:
#       type: calculate_dists
#       better: lower
```
Removing the # above still results in an error.

```
#############
# Validation
#############
# Uncomment these for validation.
val:
  val_freq: 1000
  save_img: True

  metrics:
    psnr:
      type: calculate_psnr
      crop_border: 4
      test_y_channel: false
    ssim:
      type: calculate_ssim
      crop_border: 4  # Whether to crop border during validation.
      test_y_channel: false  # Whether to convert to Y(CbCr) for validation.
    dists:
      type: calculate_dists
      better: lower
```

The above will work properly.　

```
#############
# Validation
#############
# Uncomment these for validation.
#val:
#  val_freq: 1000
#  save_img: True

#  metrics:
#    psnr:
#      type: calculate_psnr
#      crop_border: 4
#      test_y_channel: false
#    ssim:
#      type: calculate_ssim
#      crop_border: 4  # Whether to crop border during validation.
#      test_y_channel: false  # Whether to convert to Y(CbCr) for validation.
#    dists:
#      type: calculate_dists
#      better: lower
```
I think the right thing to do is this.
I apologize if I have pointed out some misunderstanding.

**Comment by the-database** [comment]
Thanks. When I use VS Code to select that text and toggle comment with `ctrl+/` it appears to work as expected with the current config files. But users who manually delete the # (which is probably most users) will run into the same issue you ran into. 

If I use your version which works better for users manually deleting the #, VS Code's toggle comment doesn't work the way it should, it removes an extra space. 

It seems a bit tricky to get this working in all scenarios. Maybe it would be better if the validation is always uncommented but there is a separate flag such as `val_enabled: true` or `false`, to avoid the need to use # at all.

**Comment by the-database** [comment]
This is addressed by #21. I added `val_enabled` and `metrics_enabled` options which should eliminate the need to comment and uncomment the validation blocks. Hope this is easier to use.
